### PR TITLE
Some fixes, some removing hacks, some adding hacks to make master images work somewhat more

### PR DESCRIPTION
**Pull request by matthew-andrews**

### CODE REVIEW

File: client/components/article/main.scss [hunk 1]
```diff
@@ -166,30 +166,6 @@ $inline-img-width: 300px;
 		width: 100%;
 	}
 }
-.article__main-image {
-	position: relative;
-	margin-top: -$brand-offset;
-	padding-bottom: 9% * 100% / 16%; // 56.25%;
```
Comment (matthew-andrews) [inline client/components/article/main.scss]: i'll re-add this logic in #198

File: client/components/article/main.scss [hunk 2]
```diff
@@ -166,30 +166,6 @@ $inline-img-width: 300px;
 		width: 100%;
 	}
 }
-.article__main-image {
-	position: relative;
-	margin-top: -$brand-offset;
```
Comment (ironsidevsquincy) [inline client/components/article/main.scss]: these was to overlap the image over the header slightly

**Comment by ironsidevsquincy** [comment]
:+1:

File: client/components/article/main.scss (hunk 2 above)
Comment (matthew-andrews) [inline client/components/article/main.scss]: i'll add that back

**Pull request merged**